--- CMakeLists.txt
+++ CMakeLists.txt
@@ -132,11 +132,8 @@
 # Global setup.
 #
 
-# For VS2022 and Xcode 12+ support.
-cmake_minimum_required(VERSION 3.21)
-
-# Only generate Debug and Release configuration types.
-set(CMAKE_CONFIGURATION_TYPES Debug Release)
+# VS2022 and Xcode 12+ support needs 3.21, but ubuntu22.04 ships with 3.20
+cmake_minimum_required(VERSION 3.20)
 
 # Project name.
 # TODO: Change this line to match your project name when you copy this file.
@@ -234,22 +231,3 @@
 
 # Display configuration settings.
 PRINT_CEF_CONFIG()
-
-
-#
-# Define the API documentation target.
-#
-
-find_package(Doxygen)
-if(DOXYGEN_FOUND)
-  add_custom_target(apidocs ALL
-    # Generate documentation in the docs/html directory.
-    COMMAND "${DOXYGEN_EXECUTABLE}" Doxyfile
-    # Write a docs/index.html file.
-    COMMAND ${CMAKE_COMMAND} -E echo "<html><head><meta http-equiv=\"refresh\" content=\"0;URL='html/index.html'\"/></head></html>" > docs/index.html
-    WORKING_DIRECTORY "${CEF_ROOT}"
-    COMMENT "Generating API documentation with Doxygen..."
-    VERBATIM )
-else()
-  message(WARNING "Doxygen must be installed to generate API documentation.")
-endif()
