--- CMakeLists.txt      2023-11-24 09:53:18.695186368 +0000
+++ CMakeLists.txt      2023-11-24 09:54:17.970959700 +0000
@@ -132,11 +132,8 @@
 # Global setup.
 #
 
-# For VS2022 and Xcode 12+ support.
-cmake_minimum_required(VERSION 3.21)
-
-# Only generate Debug and Release configuration types.
-set(CMAKE_CONFIGURATION_TYPES Debug Release)
+# VS2022 and Xcode 12+ support needs 3.21, but ubuntu22.04 ships with 3.20
+cmake_minimum_required(VERSION 3.20)
 
 # Project name.
 # TODO: Change this line to match your project name when you copy this file.
